custom_components/powercalc/data/signify/LCE002/model.json

```
@@ -4,6 +4,10 @@
     "supported_modes": [
         "lut"
     ],
+    "aliases": [
+        "929002294203",
+        "LCE002"
+    ],
```

```suggestion
    "aliases": [
        "929002294203"
    ],
```

No need to add LCE002 as this is the model id (directory name) already. Only aliases has to be listed here